tmt/utils.py

```diff
@@ -4426,6 +4426,26 @@ def remove_color(text: str) -> str:
     return re.sub(r'\x1B(?:[@-Z\\-_]|\[[0-?]*[ -/]*[@-~])', '', text)
 
 
+def git_hash(*, directory: Path, logger: tmt.log.Logger) -> Optional[str]:
+    """
+    Returned short hash of current HEAD in the git repo in directory.
+
+    :param directory: path to a local git repository.
+    :param logger: used for logging.
+    :returns: short has as string
+    """
+    cmd = Command("git", "rev-parse", "--short", "HEAD")
+    result = cmd.run(cwd=directory, logger=logger)
+
+    if result.stdout is None:
+        raise RunError(message="Empty output.",
```

Comment:
How about "git returned no output when looking for hash of the current HEAD", or something similar? Without `TMT_SHOW_TRACEBACK=1`, this text, written in red, would be the only error message emitted by tmt, and "Empty output" does not really say much about where to start digging.

Reply:
changed in eb63e98d